.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server.
+    We appreciate your time and interest in helping our project!
+
+    Please use this template to help us review your change.
+
+    Depending on your change, some sections may be unneeded, just remove these.
+    For example, refactor pull requests usually don’t need the section “Action”.
+
+    Remember that the more helpful info your pull request includes,
+    the easier it is for us to understand and review your changes.
+
+    Ensure that you’ve read through and followed the Contributing Guidelines, at
+    https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
+-->
+
+# Problem
+<!--
+    Anything that helps us understand why you are making this change goes here.
+    What problem are you trying to fix? What does this change address?
+-->
+
+:beginner: Reference related tickets with MBS-XXX at least.
+
+
+# Solution
+<!--
+    Talk about technical details, considerations, or other interesting points.
+-->
+
+:beginner: If you have a lot to say, be more detailed in this section.
+
+
+# Checklist for author
+<!--
+    The tasks you have to do to get your change ready for review. Use this if
+    you draft a pull request. Mark done tasks with an [x] as you progress. See
+    https://github.blog/2019-02-14-introducing-draft-pull-requests/
+-->
+
+* [x] Read the [Contributing Guidelines](https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md)
+* [ ] Delete irrelevant parts of this template
+* [ ] Update the [WikiDocs pages](https://wiki.musicbrainz.org/Category:WikiDocs_Page)
+  * [ ] :beginner: Page...
+* [ ] :beginner: Task...
+
+
+# Action
+<!--
+    Other than merging your change, do you want / need us to do anything else
+    with your change? This could include reviewing a specific part of your PR.
+-->
+
+1. Transclude updated WikiDocs pages to MusicBrainz documentation
```

Comment:
I have a slight feeling people aren't going to know what this means, so won't know enough to remove it if it doesn't apply, but we'll see.

Reply:
Replaced the obscure “transclude” with “make … visible … on MusicBrainz website” in https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/c5199329f6f41d128ade37536a921fedf4f7a3dd.